packages/commonwealth/server/controllers/server_comments_methods/update_comment.ts

```diff
@@ -117,6 +107,19 @@ export async function __updateComment(
     }
   })();
   await comment.save();
+
+  if (versionHistory) {
+    // The update above doesn't work because it can't detect array changes so doesn't write it to db
+    await this.models.Comment.update(
+      {
+        version_history: versionHistory,
+      },
+      {
+        where: { id: comment.id },
+      },
+    );
+  }
```

Comment:
Can you wrap both of these in a transaction?

Reply:
Turns out the comment.save will deadlock since we are making two updates to the same row in the transaction. As a result I just combined the updates into a single one for this handler